### ISSUE TITLE
Get-NavContainerSharedFolders returns incorrect values

### ISSUE THREAD

**Issue by matevzpoljanc-spin** [microsoft/navcontainerhelper#929]
Running Get-NavContainerSharedFolders returns incorrect values for file paths in container.

Versions:
Windows: 10.0.18362 Build 18362
Docker Engine: 19.03.8

Returned container paths have `:rw` at the end of path. This causes other scripts such as `Backup-NavContainerDatabases` to fail.

Most likely docker changed the way they return binds (have not done any research into this, so this is merely a speculation).

`docker inspect <container>` returns:
 ```
[
    {
     ...
     "HostConfig": {
            "Binds": [
                "C:\\ProgramData\\NavContainerHelper:C:\\ProgramData\\NavContainerHelper:rw",
            ],
            ...
     }
]
```

**Comment by freddydk** [comment]
I cannot repro that this happens automatically.
I can however share a folder with :ro, which will cause the same issue.
Will fix the function to be more recilient.

**Comment by freddydk** [comment]
Shipped in 0.6.5.6
